mineos.json

```diff
@@ -6,7 +6,9 @@
     "mount_procfs": 1,
     "mount_linprocfs": 1,
     "allow_raw_sockets": 1,
-    "boot": 1
+    "boot": 1,
```

Comment:
`boot=1` is implicit in iocage iirc. @william-gr do you think we shouldn't make it implicit there or perhaps not do it here then ?

Reply:
We should drop it.

Reply:
It shouldnt be part of the plugin property, I think

Reply:
Perfect. @miwi-fbsd can we please drop `boot` properties from plugins which define it as iocage adds them automatically.

Reply:
done